fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -1766,7 +1766,7 @@ public class Config extends ConfigBase {
      * Used to determined how many statistics collection SQL could run simultaneously.
      */
     @ConfField
-    public static int statistics_simultaneously_running_task_num = 5;
+    public static int statistics_simultaneously_running_task_num = 3;
```

Comment:
we should move all statistics paramters from config into global variables to avoid restart fe.